crates/ark/src/variables/variable.rs

```diff
@@ -1779,6 +1818,14 @@ mod tests {
         })
     }
 
+    fn inspect_from_expr(code: &str) -> Vec<Variable> {
+        let env = harp::parse_eval_global("new.env()").unwrap();
+        harp::parse_eval0(format!("x <- {code}").as_str(), env.clone()).unwrap();
```

Comment:
`Environment::bind()` would be clearer.

Reply:
Done in https://github.com/posit-dev/ark/pull/629/commits/ba5e35eedbdbd261f043f167ad9a6560b977180e